src/sentry/tagstore/base.py

```diff
@@ -391,3 +393,9 @@ def update_group_tag_key_values_seen(self, project_id, group_ids):
         >>> update_group_tag_key_values_seen(1, [2, 3])
         """
         raise NotImplementedError
+
+    def get_groups_times_seen_by_environment_name(self, project_id, group_ids, environment_name):
```

Comment:
`_by_environment_name` is to clarify that this doesn't take an `environment_id`.

Reply:
Doesn't the parameter name already make that relatively clear? Or do we already have a convention of un-prefixed methods indicating 'by_id' which we don't want here.

Reply:
Yeah the idea here is that almost all tagstore (basically where it's possible to do so) takes an `environment_id`. So this method is a definite outlier.

Reply:
This is mostly to highlight the contrast with the behavior of the majority of the other methods on this class which take the `id` and /not/ the `name`. Also willing to change this if anybody has a strong dissenting opinion